pkgs/development/python-modules/aws-sam-translator/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, boto3
+, enum34
+, jsonschema
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "aws-sam-translator";
+  version = "1.5.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9d8a25e058c78d2cef5c07aec7f98cbc2070dbfc2eb6a2e102a16beafd14e3ca";
+  };
+
+  # Tests are not included in the PyPI package
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    boto3
+    enum34
```

Comment:
enum34 should be added conditionally, that is, only with Python 2.

Reply:
I don't think this package supports Python3 at all, I need to express it somewhere

Reply:
@andreabedini  use `disabled = isPy3k;`